### ISSUE TITLE
[Bug] Can not begin to use issue

### ISSUE THREAD

**Issue by Kristinita** [SublimeLinter/SublimeLinter-pydocstyle#13]
### 1. Summary

I can not begin to use SublimeLinter-pydocstyle. I successful use 11 SublimeLinter linters, I don't have this problem.

### 2. Settings

For example, I have a file `Loverna.py`:

```python
# -*- coding: utf-8 -*-
# @Author: SashaChernykh
# @Date: 2017-04-24 06:43:14
# @Last Modified time: 2017-04-24 15:39:28
import pygame
import sys

sys.path.append('C:\Python36\Lib\site-packages\pygame')

import sublime
import sublime_plugin


class LovernaStandardCommand(sublime_plugin.TextCommand):

    def run(self, edit, seconds=10):
        self.print_seconds(sublime.active_window().new_file(), seconds)

    def print_seconds(self, view, seconds):
        pygame.mixer.init()
        text = ''
        if seconds >= 0:
            text = str(seconds)
            if seconds == 10:
                text += ' seconds'
            sublime.set_timeout_async(
                lambda: self.print_seconds(
                    view, seconds - 1), 1000)
            if seconds == 5:
                text += ' seconds'
                pygame.mixer.music.load("beep-09.mp3")
                pygame.mixer.music.play()
            if seconds == 0:
                text = 'Time over!'
                sublime.set_timeout_async(
                    lambda: view.window().run_command('force_quit'), 5000)

        view.run_command(
            'append', {
                'characters': text + '\n', 'scroll_to_end': True})

```

### 3. Steps to reproduce

I reproduce the problem in a version of Sublime Text without plugins and user settings.

I install pydocstyle, SublimeLinter and SublimeLinter-pydocstyle → I restart Sublime Text → I open file `Loverna.py` in Sublime Text.

### 4. Expected behavior

If I run pydocstyle from terminal:

```python
D:\... 3126 x64 For Debug\Data\Packages\Loverna>pydocstyle Loverna.py
Loverna.py:1 at module level:
        D100: Missing docstring in public module
Loverna.py:14 in public class `LovernaStandardCommand`:
        D101: Missing docstring in public class
Loverna.py:16 in public method `run`:
        D102: Missing docstring in public method
Loverna.py:19 in public method `print_seconds`:
        D102: Missing docstring in public method
```

### 5. Actual behavior

Nothing occurs, I don't see gutter marks, my words doesn't highlight.

If <kbd>Ctrl+Shift+P</kbd> (<kbd>⌘⇧p</kbd> for Mac) → `SublimeLinter: Enable Debug Mode` I see in Sublime Text console:

```python
SublimeLinter: pydocstyle: Loverna.py ['C:\\Python36\\Scripts\\pydocstyle.exe']
```

`C:\\Python36\\Scripts\\pydocstyle.exe` — correct path to my `pydocstyle.exe` file.

### 6. Environment

**Operating system and version:**
Windows 10 Enterprise LTSB 64-bit EN
**Sublime Text:**
Build 3126
**Package:**
I use the latest stable version of this package for Sublime Text 3
**Python:**
3.6.1
**pydocstyle:**
2.0.0

Thanks.

**Comment by brbsix** [comment]
See https://github.com/SublimeLinter/SublimeLinter-pydocstyle/issues/12. You can use https://github.com/brbsix/SublimeLinter-pydocstyle/commit/97c481fe8a81edb6a575536debd5ad17752add3a for Python 3 support.

**Comment by Kristinita** [comment]
@brbsix , maybe you make a pull request?

Thanks.

**Comment by groteworld** [comment]
Closed with the addition of #14
